src/test/typescript/component/form/Validation.spec.ts

```diff
@@ -288,6 +288,12 @@ describe("Validation", () => {
             orcid: "invalid-orcid",
             dai: "123456789",
         }
+        const contributor6: Contributor = {
```

Comment:
I would like to see more meaningful names and/or have definitions and usage more closely together.